src/patternfly/components/Page/page.scss

```diff
@@ -272,50 +280,36 @@
   margin-bottom: var(--pf-c-page__header-tools--MarginBottom);
```

Comment:
Removing `margin-top` and `margin-bottom` here doesn't seem to affect layout. Is it needed? I think originally, it was necessary for the avatar, but I can't remember.

Reply:
I think you're right, created this issue to remove it during the breaking change period: #2040